proto/v1/actuator_service.proto

```diff
@@ -16,13 +17,70 @@ service ActuatorService {
     option (google.api.method_signature) = "";
   }
 
+  rpc UpdateActuatorInfo(UpdateActuatorInfoRequest) returns (ActuatorInfo) {
+    option (google.api.http) = {
+      patch: "/v1/actuator/info"
+      body: "actuator"
+    };
+    option (google.api.method_signature) = "actuator,update_mask";
+  }
+
   rpc DeleteCache(DeleteCacheRequest) returns (google.protobuf.Empty) {
     option (google.api.http) = {delete: "/v1/actuator/cache"};
   }
+
+  rpc ListDebugLog(ListDebugLogRequest) returns (ListDebugLogResponse) {
+    option (google.api.http) = {get: "/v1/actuator/debug"};
+    option (google.api.method_signature) = "";
+  }
 }
 
 message GetActuatorInfoRequest {}
 
+message UpdateActuatorInfoRequest {
+  // The actuator to update.
+  ActuatorInfo actuator = 1 [(google.api.field_behavior) = REQUIRED];
+
+  // The list of fields to update.
+  google.protobuf.FieldMask update_mask = 2 [(google.api.field_behavior) = REQUIRED];
+}
+
+message ListDebugLogRequest {
+  // The maximum number of logs to return. The service may return fewer than
+  // this value.
+  // If unspecified, at most 50 logs will be returned.
+  // The maximum value is 1000; values above 1000 will be coerced to 1000.
+  int32 page_size = 1;
+
+  // A page token, received from a previous `ListDebugLog` call.
+  // Provide this to retrieve the subsequent page.
+  //
+  // When paginating, all other parameters provided to `ListDebugLog` must match
+  // the call that provided the page token.
+  string page_token = 2;
+}
+
+message ListDebugLogResponse {
+  // The logs from the specified request.
+  repeated DebugLog logs = 1;
+
+  // A token, which can be sent as `page_token` to retrieve the next page.
+  // If this field is omitted, there are no subsequent pages.
+  string next_page_token = 2;
+}
+
+message DebugLog {
+  int64 record_ts = 1;
+
+  string request_path = 2;
+
+  string role = 3;
+
+  string error = 4;
+
+  string stack_trace = 5;
+}
```

Comment:
PTAL the proto @d-bytebase. I plan to
- Remove the `method` filed in `DebugLog` and only keep the `request_path`, as in gRPC we will get the full method path like `/bytebase.v1.ActuatorService/GetActuatorInfo`
- Put the `debug` flag in the actuator

Reply:
SGTM